### PR TITLE
nef about - app

### PR DESCRIPTION
**Pull request by miguelangel-dev**
## Description
We want to use the power of `nef` in Xcode - in last [PR](https://github.com/bow-swift/nef-plugin/pull/1) we create the plugin for Xcode where user can export the selected code using `nef` (and Carbon integration). In the [next one](https://github.com/bow-swift/nef-plugin/pull/2)  we completed the second option, the `Carbon preferences` for customization how the selected code will be exported.

In this PR, we will provide the `About` page where the user will see how to install the plugin.

## How did it implement?
`About option` has been done using `SwiftUI`

## Screenshots
Captura de pantalla 2019-08-05 a las 16 37 05

[nef.zip](https://github.com/bow-swift/nef-plugin/files/3468118/nef.zip)

### CODE REVIEW

File: nef/About/AboutView.swift [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        static let preferences = Image("system-preferences")
+        static let extensions = Image("system-extensions")
+    }
+    
+    enum Constants {
+        static let githubURL = "https://github.com/bow-swift/nef#-features"
+    }
+    
+    enum i18n {
+        static let copyright = NSLocalizedString("Copyright © 2019 The nef authors", comment: "")
```
Comment (truizlop) [inline nef/About/AboutView.swift]: I think `NSLocalizedString` is no longer needed when working with SwiftUI. Check [this](https://nathfreder.dev/localizing-your-swiftui-app/).

**Review by truizlop — APPROVED** [review]

File: nef/About/AboutView.swift (hunk 1 above)
Comment (miguelangel-dev) [reply nef/About/AboutView.swift]: uh! good point, I did not know it will go auto - I will change it

Comment (miguelangel-dev) [reply nef/About/AboutView.swift]: I have reviewed it and it only works if we use 'harcoded key' directly in the constructor, for example:

`Toggle('preferences_key', isOn: $state.value)`
`Text('preferences_key')`
`Image('preferences_image')`

But if we want for example reuse the same key in different elements, and prevent to use 'hardcoded' strings in the code, we can not do it:

`Toggle(Const.preferencesKey, isOn: $state.value)`
`Text(Const.preferencesKey)`

Const.PreferencesKey = 'preferences_key'

So, I think it is better to continue using `NSLocalizedString` in favour of reusability. Wdut @truizlop ?

Comment (miguelangel-dev) [reply nef/About/AboutView.swift]: On another hand, I have localized the plugin (en|es)

Captura de pantalla 2019-08-12 a las 17 06 37

Comment (truizlop) [reply nef/About/AboutView.swift]: Ok, it's good to know that.

**Review by truizlop — APPROVED** [review]

**Pull request merged**